src/api/app/models/workflow/step.rb

```diff
@@ -158,4 +144,8 @@ def validate_project_and_package_name
   def webhook_event_for_linking_or_branching?
     scm_webhook.new_pull_request? || (scm_webhook.updated_pull_request? && target_package.blank?) || scm_webhook.push_event? || scm_webhook.tag_push_event?
   end
+
+  def scm_event_subscription
+    @scm_event_subscription ||= Workflows::ScmEventSubscription.new(token, workflow_run, scm_webhook)
```

Comment:
What do you gain by memoization of this service?

Reply:
Because of [this](https://github.com/openSUSE/open-build-service/pull/14668/files#diff-5e1c6eb71502c591dca7a382d703c9453fd4771a7d824a4ad0f53b3d51037cf2R160), but now I realized that it needs fixing...